teuthology/packaging.py

```diff
@@ -472,9 +472,13 @@ def _init_from_remote(self):
         self.os_version = self.remote.os.version
         self.codename = self.remote.os.codename
         self.pkg_type = self.remote.system_type
+        _version = self.remote.os.version
+        _major_version = self.job_config.get('install_per_major_version', False)
```

Comment:
Just planing ahead, but I don't think this will work for SUSE [1] since it stores its version as `15.2`. Perhaps this strip the minor logic just needs to be conditional on `rhel`/`centos` (similar to how ubuntu/debian switches the version to code names like `bionic`).

[1] https://shaman.ceph.com/api/search/?status=ready&project=ceph&ref=master&distros=opensuse/15.2&sha1=latest

Reply:
Yeah, right.
I will close the PR now, because the `_get_distro()` code has already done the minor version stripping for centos/rhel.